test/Microsoft.AspNetCore.Server.Kestrel.FunctionalTests/RequestTests.cs

```diff
@@ -76,6 +80,72 @@ public class RequestTests
             }
         }
 
+        [Fact]
+        public async Task LargeMultipartUpload()
+        {
+            var port = PortManager.GetPort();
+            var config = new ConfigurationBuilder()
```

Comment:
You don't need to do this anymore. There's a UseUrl extension method hanging off `WebHostBuilder`